hudi-platform-service/hudi-metaserver/hudi-metaserver-client/src/main/java/org/apache/hudi/common/table/HoodieTableMetaserverClient.java

```diff
@@ -63,15 +60,12 @@ public HoodieTableMetaserverClient(Configuration conf, String basePath, Consiste
                                      String databaseName, String tableName, HoodieMetaserverConfig config) {
     super(conf, basePath, false, consistencyGuardConfig, Option.of(TimelineLayoutVersion.CURR_LAYOUT_VERSION),
         config.getString(HoodieTableConfig.PAYLOAD_CLASS_NAME), mergerStrategy, fileSystemRetryConfig);
-    checkArgument(nonEmpty(databaseName), "database name is required.");
-    checkArgument(nonEmpty(tableName), "table name is required.");
```

Comment:
Instead of null, can we change the param type as an Option ?

Reply:
It has been updated according to your review changes.